runtime/src/iree-amd-aie/aie_runtime/iree_aie_runtime.h

```diff
@@ -294,6 +294,11 @@ struct AMDAIEDeviceModel {
 
   uint32_t getColumnShift() const;
   uint32_t getRowShift() const;
+
+  /// Return a map from channels to valid BD ids for the requested tile type.
+  /// TODO(jornt): find these ranges in the device model.
```

Comment:
it's here https://github.com/Xilinx/aie-rt/blob/c28d57972957d8032839f4da5d2e3b9dd01805ea/driver/src/global/xaie2ipugbl_reginit.c#L389-L391 but they don't distinguish between even/odd

Reply:
Yeah, they have the real numbers harcoded inside a verifier here: https://github.com/Xilinx/aie-rt/blob/1ad203de0b7f282b1c0659fd2ae3f218652c7274/driver/src/dma/xaie_dma_aieml.c#L1216

Reply:
We can send them a PR to factor that out (but also that's a good util to use somewhere around here too)